server/api/school-invite/school-invite.controller.js

```diff
@@ -124,6 +131,52 @@ class SchoolInvite {
       return false
     }
   }
+
+  static async accept (request, response) {
+    return handleToken(request, response, {
+      join: async (props) => {
+        const organisation = await SchoolInvite.createOrganisationFromSchool(props.schoolId)
+        await SchoolInvite.linkPersonToOrganisationAsAdmin(organisation._id, request.session.me._id)
+      }
+    })
+  }
+
+  static async createOrganisationFromSchool (schoolId) {
+    const schoolData = await SchoolLookUp.findOne({ schoolId: schoolId })
```

Comment:
perhaps should check here that you got a result. I presume schoolId originated from the db lookup so should not fail.
However the db could have other issues so you'll want to check

Reply:
Yep makes sense - I'll add that now